src/components/anchor-navigation/anchor-navigation-item.d.ts

```diff
@@ -14,6 +14,8 @@ export interface AnchorNavigationItemProps {
   isSelected: boolean;
   /** Allows to override existing component styles */
   styleOverride?: () => object | object;
+  /** Reference to the section html element meant to be shown   */
+  target?: HTMLElement;
```

Comment:
```suggestion
  target?: RefObject<HTMLElement>;
```

Shouldn't this be a `RefObject`?